### ISSUE TITLE
Creating custom theme

### ISSUE THREAD

**Issue by abdessalaam** [cweijan/vscode-office#382]
Hi, I would like to be able to create a custom theme, with font-family, margins, colours etc. The reason is that I use markdown to publish to my website, and it would be great to be able to match the theme of my website so that when I am writing I can see what it will look like eventually.  

Would this be possible?  

Thank you!

**Comment by cweijan** [comment]
No, the theme files are complex, and I won't be creating a feature specifically for that. Version 4.0.5 has refactored the Markdown editor, providing you with a better experience now.
